src/Module/Contact/Poke.php

```diff
@@ -44,14 +44,14 @@ public static function post(array $parameters = [])
 
 		Logger::info('verb ' . $verb . ' contact ' . $contact_id);
 
-		$contact = DBA::selectFirst('contact', ['id', 'name'], ['id' => $parameters['id'], 'uid' => local_user()]);
+		$contact = DBA::selectFirst('contact', ['id', 'name', 'url'], ['id' => $parameters['id'], 'uid' => local_user()]);
 		if (!DBA::isResult($contact)) {
 			return self::postReturn(false);
 		}
 
 		$a = DI::app();
 
-		$private = (!empty($_GET['private']) ? intval($_GET['private']) : Model\Item::PUBLIC);
+		$private = !empty($_POST['private']) ? Model\Item::PRIVATE : Model\Item::PUBLIC;
```

Comment:
Why not `_REQUEST` to handle both `_POST` and `_GET`?

Reply:
I just saw, it is in the `post` section, so `_POST` is fin.

Reply:
Because the rest of the function uses `$_POST` and I didn’t want to spend too much time on a feature I believe should be removed.